server/src/main/java/com/google/sps/utils/ReportReader.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Author: Kira Toal
+ * Date: June 24, 2020
+ */
+package com.google.sps.utils;
+
+import com.google.sps.utils.Ad;
+import com.google.sps.utils.AdRowProcessor;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+
+public class ReportReader {
+
+  private static final String CSV_FILE_PATH = "/output.csv";
+  private static final String COLLECTION = "testing"; 
+  private static final int START_ROW_INDEX = 1;  // first csv row that gets read in to DB (1 based)
```

Comment:
Is this because it has a header row?  Take a look and see if you can specify that in the parameters to InputStreamReader.

Also specify whether this is inclusive or exclusive.